### ISSUE TITLE
Can this code generate background noise similar to scanned images(300dpi)?

### ISSUE THREAD

**Issue by aiFTong** [Belval/TextRecognitionDataGenerator#80]
i want to generate training data for text recognition of sacnned images. 

Using the generated data, high accuracy can be achieved in the training set, but when I tested it on real scanned images, it didn't work well.

I wonder if the generated data might not be quite the same as the real scanned data, Can this code generate background noise similar to scanned images(300dpi)?

Thank you in advance for your help!

**Comment by Belval** [comment]
There's no feature for that in the project right now, but I would be interested in adding it. Can you provide sample of the type of noise that would resemble scanning artifacts?

Also, in the short term you can:

- Look up [imgaug](https://github.com/aleju/imgaug) to help with augmentations that could achieve similar results. 
- Use different background types (see quasicrystals) and play with the different levels of gaussian blur ([see the doc](https://textrecognitiondatagenerator.readthedocs.io/en/latest/overview.html#getting-help)).

Finally, do check that your model isn't over fitting. Adding dropout in some layers and regularization could be a good way to handle this.
